app/src/main/res/layout/fragment_product_detail.xml

```diff
@@ -22,224 +23,291 @@
-                            android:onClick="@{() -> viewModel.onClickSeeAllProduct()}"
-                            android:text="@string/see_all"
-                            android:textColor="@color/color_orange_mos"
-                            android:textStyle="bold"
-                            tools:ignore="RelativeOverlap"
-                            />
-                    </RelativeLayout>
+                        android:drawableLeft="@drawable/ic_label"
+                        android:drawablePadding="@dimen/dp_5"
+                        android:drawableStart="@drawable/ic_label"
+                        android:gravity="center_vertical"
+                        android:text="@string/products_shop"
+                        android:textColor="@color/dark_gray"
+                        android:textStyle="bold"
+                        />
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentEnd="true"
+                        android:layout_alignParentRight="true"
+                        android:layout_centerVertical="true"
+                        android:layout_marginEnd="@dimen/dp_20"
+                        android:layout_marginRight="@dimen/dp_20"
+                        android:gravity="center_vertical"
+                        android:onClick="@{() -> viewModel.onClickSeeAllProduct()}"
+                        android:text="@string/see_all"
+                        android:textColor="@color/color_orange_mos"
+                        android:textStyle="bold"
+                        tools:ignore="RelativeOverlap"
+                        />
                 </RelativeLayout>
-                <android.support.v7.widget.RecyclerView
-                    android:id="@+id/recycle_view_productInShop"
+                <FrameLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_margin="@dimen/dp_2"
-                    android:layout_marginTop="@dimen/dp_5"
-                    android:orientation="horizontal"
-                    android:padding="@dimen/dp_2"
-                    app:layoutManager="LinearLayoutManager"
-                    app:recyclerAdapter="@{viewModel.productShopAdapter}"
-                    />
+                    >
+                    <android.support.v7.widget.RecyclerView
+                        android:id="@+id/recycle_view_productInShop"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_margin="@dimen/dp_2"
+                        android:layout_marginTop="@dimen/dp_5"
+                        android:orientation="horizontal"
+                        android:paddingEnd="@dimen/dp_3"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:paddingRight="@dimen/dp_3"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.